.github/workflows/ci.yml

```diff
@@ -255,6 +235,8 @@ jobs:
         with:
           path: src/github.com/containerd/containerd
 
+      - uses: ./.github/actions/install-go
```

Comment:
We just need checkout path and uses to align here.

Maybe also #9946 while you have the hood open?

Reply:
Yes, We can club these changes together.

Reply:
Done. added the cherry-pick